### PR TITLE
Bugfix/force authn

### PR DESCRIPTION
**Pull request by harrdou**

### CODE REVIEW

File: gluu-server/opt/gluu/jetty/oxauth/custom/scripts/person_authentication/SignInCanada.py [hunk 1]
```diff
@@ -455,7 +465,7 @@ def getPageForStep(self, configurationAttributes, step):
                 # The missing file extension below is intentional.
                 # This forces an extra redirect so that the request URL matches a rule in the
```
Comment (harrdou) [inline gluu-server/opt/gluu/jetty/oxauth/custom/scripts/person_authentication/SignInCanada.py:466]: These comments no longer apply and can be removed.

Comment (innovatechs) [reply gluu-server/opt/gluu/jetty/oxauth/custom/scripts/person_authentication/SignInCanada.py]: Good point Doug. Thanks

**Review by vid — CHANGES_REQUESTED** [review]
Can we fix the build? Corretto is failing to install.

**Review by innovatechs — APPROVED** [review]

**Comment by harrdou** [comment]
> Can we fix the build? Corretto is failing to install.

Fixed. See AB#7735.

The SKS keyservers were shut down last week.

**Comment by vid** [comment]
Looks like we have some "input must be valid" errors, guessing you are on it.

**Pull request merged**